app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,18 @@
+<% provide(:title, "About") %>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= yield(:title) %> <%= t".title" %></title>
+  </head>
+  <body>
+    <h1><%=t ".about" %></h1>
+    <p>
+      The <a href="http://www.railstutorial.org/"><em><%=t ".rubyonrails" %></em></a> <%=t ".is" %>
```

Comment:
i18n cho The
<%= t ".rubyonrails" %> trước t có 1 space. em fix tất cả những chỗ khác. lỗi này em đang bị rất nhiều